Sources/UnleashProxyClientSwift/Poller.swift

```diff
@@ -168,3 +172,7 @@ public class Poller {
         })
     }
 }
+
+fileprivate extension CharacterSet {
+    static let rfc3986Unreserved = CharacterSet(charactersIn: "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~")
```

Comment:
`CharacterSet.urlQueryAllowed` by default allows `+` (meaning space), this adds more strict percent encoding to handle this case.

https://stackoverflow.com/a/41568692